src/Elliptic/Executables/Poisson/SolvePoissonProblem.hpp

```diff
@@ -71,6 +74,13 @@ struct Metavariables {
   using analytic_solution_tag =
       Tags::AnalyticSolution<Poisson::Solutions::ProductOfSinusoids<Dim>>;
 
+  // The initial guess for the elliptic solve. Set to the `analytic_solution`
+  // if it provides an initial guess, or to `elliptic::NumericInitialGuess`
+  // to load data from disk. With #1812 the initial guess will be chosen when
+  // setting up the executable instead.
+  using initial_guess = tmpl::type_from<analytic_solution_tag>;
+  // using initial_guess = elliptic::NumericInitialGuess<system>;
+
```

Comment:
Please merge #1812 and fix this before merging so that you actually build and test numerical import in an executable